src/extensions/default/CSSCodeHints/main.js

```diff
@@ -175,16 +175,20 @@ define(function (require, exports, module) {
         return true;
     };
 
-    /*
+    /**
      * Returns a sorted and formatted list of hints with the query substring
      * highlighted.
      * 
      * @param {Array.<Object>} hints - the list of hints to format
      * @param {string} query - querystring used for highlighting matched
-     *      poritions of each hint
+     *      portions of each hint
      * @return {Array.jQuery} sorted Array of jQuery DOM elements to insert
      */
     function formatHints(hints, query) {
+        var hasColorSwatches = hints.some(function (token) {
```

Comment:
[Array.prototype.some()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some?redirectlocale=en-US&redirectslug=JavaScript%2FReference%2FGlobal_Objects%2FArray%2Fsome) - "`some` executes the `callback` function once for each element present in the array until it finds **one** where `callback` returns a `true` value. If such an element is found, `some` immediately returns `true`. Otherwise, `some` returns `false`."